legal-api/src/legal_api/services/filings/validations/restoration.py

```diff
@@ -30,31 +30,29 @@
 APPROVAL_TYPE_PATH = "/filing/restoration/approvalType"
 
 
-def validate(legal_entity: LegalEntity, restoration: Dict) -> Optional[Error]:
+def validate(business: any, restoration: Dict) -> Optional[Error]:
     """Validate the Restoration filing."""
     filing_type = "restoration"
-    if not legal_entity or not restoration:
+    if not business or not restoration:
         return Error(HTTPStatus.BAD_REQUEST, [{"error": babel("A valid business and filing are required.")}])
     msg = []
 
     restoration_type = get_str(restoration, "/filing/restoration/type")
     limited_restoration = None
     if restoration_type in ("limitedRestorationExtension", "limitedRestorationToFull"):
         limited_restoration = Filing.get_a_businesses_most_recent_filing_of_a_type(
```

Comment:
Unrelated question here. Will this method support alternate name in the future?

Reply:
yes, i think it should